c_emulator/riscv_sim.c

```diff
@@ -133,6 +133,8 @@ static struct option options[] = {
 #ifdef SAILCOV
   {"sailcov-file",                required_argument, 0, 'c'},
 #endif
+  {"zicbom-cache-block-size",            required_argument, 0, 'M'},
```

Comment:
I don't think there should be multiple options here but rather a single `--cache-block-size` option.

As the spec says: `In the initial set of CMO extensions, the
size of a cache block shall be uniform throughout the system.` so it really doesn't make sense to have different sizes of Zicboz and Zicbom.